batch-setup/make_meta_tiles.py

```diff
@@ -457,15 +484,25 @@ def render(self, num_retries, lense):
     assert args.metatile_size > 0
     assert args.metatile_size < 100
     metatile_max_zoom = 16 - metatile_zoom_from_size(args.metatile_size)
+
+    generator = None
+    if args.use_tiles_coords_generator:
+        generator = BoundingBoxTilesCoordinateGenerator(west=-4.1494623,
```

Comment:
Please set the default bbox to be over Seattle, SF, or NYC.

Reply:
> Please set the default bbox to a wide region over Seattle, SF, or NYC.

removed the hardcoded bbox